### ISSUE TITLE
Integrate web crawler with CI

### ISSUE THREAD

**Issue by psmyrek** [ckeditor/ckeditor5#8801]
## Provide a description of the task

Integrate the web crawler introduced in PR https://github.com/ckeditor/ckeditor5/pull/8745 with the CI process.

[This improvement](https://github.com/ckeditor/ckeditor5/issues/8800) should be done before the integration with CI to get rid off false positives.

**Comment by pomek** [comment]
Closing in favor of 645 in the private issue tracker.
